src/Agent.Plugins/PipelineCache/SavePipelineCacheV0.cs

```diff
@@ -51,13 +50,27 @@ public class SavePipelineCacheV0 : PipelineCacheTaskPluginBase
                 return;
             }
 
+            await base.RunAsync(context, token);
+        }
+
+        protected override async Task ProcessCommandInternalAsync(
+            AgentTaskPluginExecutionContext context,
+            Fingerprint fingerprint,
+            Func<Fingerprint[]> restoreKeysGenerator,
+            string path,
+            CancellationToken token)
+        {
+            VariableValue packValue = context.Variables.GetValueOrDefault(PackingVariableName);
+            string pack = packValue?.Value ?? string.Empty;
+            string contentFomat = (!String.IsNullOrWhiteSpace(pack)) ? ContentFormatConstants.SingleTar : ContentFormatConstants.Files;
```

Comment:
😱

Reply:
Yeah - just about anything, just need that variable to be not null :) , reasoning : we will be removing that soon.